template/base.html.jinja

```diff
@@ -57,7 +57,8 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@docsearch/css@3" />
 
     <script src="https://cmp.osano.com/AzyjT6TIZMlgyLyy8/f11f7772-8ed5-4b73-bd17-c0814edcc440/osano.js"></script>
-    <link rel="stylesheet" href="https://www.unpkg.com/@xrpl/ai-css/xrplai.css" />
+    <!-- <link rel="stylesheet" href="https://www.unpkg.com/@xrpl/ai-css/xrplai.css" /> -->
+    <link rel="stylesheet" href="{{currentpage.prefix}}assets/css/xrplai.css" />
```

Comment:
Why is this link commented? (Should it be reverted to the old link, or should that comment be deleted?)

Reply:
this would be brought back once the changes on xrplai is deployed

Reply:
Since this PR is not in draft mode - can you change it to the way it should be merged? (Locally you can have it differently and just not commit in order to test changes)

Otherwise I'm uncomfortable approving the PR because this is not what should be merged